app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -27,17 +28,24 @@ const Entry = ({
 }: {
   params: { leagueId: string };
 }): JSX.Element => {
-  const [entries, setEntries] = useState<IEntry[]>([]);
   const [currentWeek, setCurrentWeek] = useState<IGameWeek['week']>(1);
+  const [entries, setEntries] = useState<IEntry[]>([]);
+  const [loadingData, setLoadingData] = useState<boolean>(true);
   const { user } = useDataStore((state) => state);
 
   /**
    * Fetches all entries for the current user.
    * @returns {Promise<void>}
    */
   const getAllEntries = async (): Promise<void> => {
-    const getEntries = await getCurrentUserEntries(user.id, leagueId);
-    setEntries(getEntries);
+    try {
+      const getEntries = await getCurrentUserEntries(user.id, leagueId);
+      setEntries(getEntries);
+    } catch (error) {
+      console.error(error);
```

Comment:
This should throw an error when entries are not retrieved.

Reply:
Added a try-catch to show error

Reply:
It needs to `throw new Error(error)`. By throwing, it will error out the component and the error boundary will catch it.

Reply:
Added in `throw new Error("Error fetching user's entries");`

Reply:
Removed the console.error